source/app/blueprints/pages/manage/manage_webhooks_routes.py

```
@@ -0,0 +1,283 @@
+    return render_template('manage_webhooks.html', form=form)
+
+
+@manage_webhooks_blueprint.route('/manage/webhooks/list', methods=['GET'])
+@ac_api_requires()
+def list_webhooks():
+    """Show a list of webhooks
+
+    Returns:
+        Response: List of webhooks
+    """
+    case_id = request.args.get("cid")  # Use 'cid' since it's coming from query parameters
```

Ruff (0.8.2)

Local variable `case_id` is assigned to but never used

Remove assignment to unused variable `case_id`

(F841)